src/pytest_ansible_network_integration/defs.py

```diff
@@ -290,9 +291,14 @@ def get_dhcp_lease(self, current_lab_id: str) -> str:
         macs = self._extract_macs(current_lab)
         logger.info("Found MAC addresses: %s", macs)
 
-        ips = self._find_dhcp_lease(macs, 100)
+        ips = self._find_dhcp_lease(macs, 200)
```

Comment:
Do we want to change this?

Reply:
Yes, we are just increasing this 200 because we would like to wait for more time for finding ip. iosxr takes atleast 10 minutes to boot hence made this change, this will not affect the time taken to start test for other devices, it just gives more time for iosxr!